package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "til-web",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "lint:fix": "next lint --fix",
+    "stylelint": "stylelint '**/*.css'",
```

Comment:
I was wondering if we have any plan for [sass](https://nextjs.org/docs/messages/install-sass), if yes maybe we can introduce it here?